### PR TITLE
Create Temporal Resolution range

### PR DESCRIPTION
**Pull request by UbuntuScott**
new DES vocab for Temporal Resolutions
@DavidCrosswellGSQ  for technical check 
@KellyVance  for content as vocab owner

### CODE REVIEW

File: vocabularies/temporalresolutionrange-with-top-concepts.ttl [hunk 1]
```diff
@@ -0,0 +1,204 @@
+<https://gcmdservices.gsfc.nasa.gov/kms/concept/40e09855-fb48-4a7d-9851-d6e809e6c309>
+        a       <http://www.w3.org/2004/02/skos/core#Concept> ;
+        <http://www.w3.org/2004/02/skos/core#broader>
+                <https://gcmdservices.gsfc.nasa.gov/kms/concept/502ad03c-dba8-4b32-8af5-13fa947c3988> ;
+        <http://www.w3.org/2004/02/skos/core#inScheme>
+                <https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/temporalresolutionrange> ;
+        <http://www.w3.org/2004/02/skos/core#prefLabel>
+                "Annual"@en .
```
Comment (KellyVance) [inline vocabularies/temporalresolutionrange-with-top-concepts.ttl:8]:
```suggestion
                "Annual xxxxxxxx"@en .
```

**Review by KellyVance — CHANGES_REQUESTED** [review]
I want the annual label changed

File: vocabularies/temporalresolutionrange-with-top-concepts.ttl (hunk 1 above)
Comment (UbuntuScott) [reply vocabularies/temporalresolutionrange-with-top-concepts.ttl:8]: Not today thanks! @KellyVance

**Review by KellyVance — APPROVED** [review]
All Good.

**Review by DavidCrosswellGSQ — APPROVED** [review]
Approved for today's meeting.
However, recommend that we insert this into GraphDB from the source vocab.

**Pull request merged**